image/cli/app-root/src/finalizer.py

```diff
@@ -350,9 +350,11 @@ def getcp4dCompsVersions():
                         if "mas.ibm.com/buildId" in deploymentDict["metadata"]["labels"]:
                             productBuildId = deploymentDict["metadata"]["labels"]["mas.ibm.com/buildId"]
                             productBuildNumber = deploymentDict["metadata"]["labels"]["mas.ibm.com/buildNumber"]
+                            productCommitId = deploymentDict["metadata"]["labels"]["mas.ibm.com/commitId"]
```

Comment:
should we check if commitId is in the labels?

Reply:
I have updated.